modules/distr-tracing-tempo-install-tempostack-cli.adoc

```diff
@@ -0,0 +1,169 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-install-tempostack-cli_{context}"]
+= Installing a TempoStack instance by using the CLI
+
+You can install a TempoStack instance from the command line.
```

Comment:
Same question applies to TempoStack globally, should it be in backticks?

Reply:
I wouldn't add back ticks to every mention of TempoStack product. I only use back ticks when referring to the custom resource and it's contents. It's like because it's possible to open Firefox by running the `firefox` command, should we write `fitefox` rather than "Firefox" everywhere. IMO, the style guide rules should be friends with the common sense.